go/pkg/bertyaccount/service_account.go

```diff
@@ -393,10 +406,13 @@ func (s *service) getAccountMetaForName(accountID string) (*AccountMetadata, err
 	return meta, nil
 }
 
-func (s *service) DeleteAccount(_ context.Context, request *DeleteAccount_Request) (*DeleteAccount_Reply, error) {
+func (s *service) DeleteAccount(ctx context.Context, request *DeleteAccount_Request) (_ *DeleteAccount_Reply, err error) {
 	s.muService.Lock()
 	defer s.muService.Unlock()
 
+	_, _, endSection := tyber.Section(ctx, s.logger, fmt.Sprintf("Deleting account %s (AccountService)", request.AccountID))
+	defer func() { endSection(err, "") }()
+
 	if s.initManager != nil {
 		return nil, errcode.ErrBertyAccountAlreadyOpened
```

Comment:
is the err in `endSection` set to `errcode.ErrBertyAccountALreadyOpened` here? or is it just returned like this with the `endSection(nil, "")`

Reply:
no, see this playground https://play.golang.org/p/ejXulLs39tt